app/view/signin/signin.html

```diff
@@ -0,0 +1,22 @@
+<ramble-nav-bar></ramble-nav-bar>
+<div class="signin-page">
+  <section class="about-splash-signin">
+    <div>
+      <h1>Ramble</h1>
+      <h2>write on...</h2>
+    </div>
+  </section>
+  <form novalidate ng-init="signinCtrl.getToken()" ng-submit="signinCtrl.signin()" name="signinForm" class="form-auth">
+    <h3>SIGN IN</h3>
+    <label for="email">EMAIL</label>
+    <input placeholder="email" class="form-control" name="email" type="email" ng-model="signinCtrl.user.username" required/>
+    <br/>
+    <label for="password">PASSWORD</label>
+    <input placeholder="password" class="form-control" name="password" type="password" ng-model="signinCtrl.user.password" required/>
+    <button ng-disabled="signinForm.email.$error.required || signinForm.password.$error.required" type="submit">Sign in</button>
```

Comment:
this is a great us of using ng-disabled so that you have to do less error handling in the javascript for this controller :)